sdk/test-utils/recorder-new/src/utils/relativePathCalculator.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import path from "path";
+import fs from "fs";
+import { RecorderError } from "./utils";
+
+export function relativeRecordingsPathForNode(testContext: Mocha.Test) {
+  const testAbsolutePath = testContext.file;
+  if (!testAbsolutePath) {
+    throw new RecorderError("Unable to grab the file path from the test run");
+  }
+
+  const recordingsPath = findRecordingsFolderPath(testAbsolutePath);
+  const recordingsFolder = path.basename(recordingsPath);
+  const projectFolder = path.basename(path.resolve(recordingsPath, ".."));
+  const serviceFolder = path.basename(path.resolve(recordingsPath, "..", ".."));
+  const sdk = path.basename(path.resolve(recordingsPath, "..", "..", ".."));
+  if (sdk !== "sdk") {
+    throw new Error("Unexpected location for recordings, please fix the location.");
+  }
+  return path
```

Comment:
Does `path.join(...args).join(path.sep)` produce anything other than args?

Reply:
> path.join(...args).join(path.sep)

Double join?

Reply:
Sorry about that, meant `path.join(...args).split(path.sep);`

Reply:
Just the args